alchemist/src/orchid/resources/wiki/usage/run.md

```diff
@@ -0,0 +1,46 @@
+---
+
+title: "Running Alchemist simulations"
+
+---
+
+Alchemist expects a YAML map as input, such maps are contained in *.yml files and are used to write simulations. Informations about how to write alchemist simulations can be found in the next section.
+
+## Running a simulation
+
+The typical way of running alchemist simulations is by directly using alchemist <kbd>main</kbd> method. If you have cloned the Alchemist repository, you can also run a simulation using the produced artifacts (namely, a .jar file). The first approach is recommended.
```

Comment:
Very unclear. What do you mean?
How do you call a method? :)

We should ideally provide a template repository on github with the Gradle wrapper and Alchemist in the dependencies, and a minimal, working simulation, and say "ok, launch `./gradlew runSimName` and it automagically runs this and that

Reply:
I had confused different things, consider it a draft. 

There's now a section explaining the user how to use the simulator via Gradle and another one explaining how to run it stand-alone